crates/fluence-app-service/src/service.rs

```diff
@@ -172,25 +174,27 @@ impl AppService {
 
     /// Return statistics of Wasm modules heap footprint.
     /// This operation is cheap.
-    pub fn module_memory_stats(&self) -> MemoryStats<'_> {
+    pub fn module_memory_stats(&mut self) -> MemoryStats<'_> {
         self.marine.module_memory_stats()
     }
 }
+#[cfg(feature = "raw-module-api")]
+use marine_wasm_backend_traits::WasiState;
```

Comment:
Shouldn't it be moved to the top?   Or, at least, separated by an empty line.

Reply:
Its hard to tell. I moved it to the part right after the common imports